Cargo.toml

```diff
@@ -25,6 +25,7 @@ serde = "1.0.118"
 serde_derive = "1.0.118"
 simplelog = "0.9.0"
 uom = { version = "0.30.0", features = ["autoconvert", "f32", "si"] }
+xcb = { version = "0.8", features = ["xkb"] }
```

Comment:
Hmmmm. I've tried to find an alternative, but it seems that all other xcb/xkb bindings are even more outdated and/or unmaintained. 

So if this is a problem I'd better maintain my own fork of dwm-status because I think I'd hardly need any other functionality/updates. I don't even really need the xkb functionality because I can use an utility like gxcb instead - I just hate to run extra programs when a single program can handle everything and besides slstatus have this functionality built-in.

Also, it was more like an attempt to prove that I can do something useful in Rust already. :)

Reply:
If it is so, I'm fine with this crate, but please use the latest version if possible :)